src/__tests__/medications/search/ViewMedications.test.tsx

```diff
@@ -39,45 +40,31 @@ describe('View Medications', () => {
       user: { permissions },
       medications: { medications: [{ ...expectedMedication, ...medication }] },
     } as any)
-    const titleSpy = jest.spyOn(titleUtil, 'useUpdateTitle').mockImplementation(() => jest.fn())
+    jest.resetAllMocks()
     const setButtonToolBarSpy = jest.fn()
-    jest.spyOn(MedicationRepository, 'search').mockResolvedValue([])
+    jest.spyOn(MedicationRepository, 'search').mockResolvedValue(givenMedications)
     jest.spyOn(ButtonBarProvider, 'useButtonToolbarSetter').mockReturnValue(setButtonToolBarSpy)
     jest.spyOn(MedicationRepository, 'findAll').mockResolvedValue([])
 
-    await act(async () => {
-      wrapper = await mount(
+    return {
+      history,
+      setButtonToolBarSpy,
+      ...render(
         <Provider store={store}>
           <Router history={history}>
             <TitleProvider>
               <ViewMedications />
             </TitleProvider>
           </Router>
         </Provider>,
-      )
-    })
-    wrapper.update()
-    return {
-      wrapper: wrapper as ReactWrapper,
-      history,
-      titleSpy,
-      setButtonToolBarSpy,
+      ),
     }
   }
 
-  describe('title', () => {
-    it('should have called the useUpdateTitle hook', async () => {
-      const { titleSpy } = await setup({} as Medication)
-
-      expect(titleSpy).toHaveBeenCalled()
-    })
-  })
-
   describe('button bar', () => {
     it('should display button to add new medication request', async () => {
       const permissions = [Permissions.ViewMedications, Permissions.RequestMedication]
-      const { setButtonToolBarSpy } = await setup({} as Medication, permissions)
-
+      const { setButtonToolBarSpy } = await setup({ medication: 'test' } as Medication, permissions)
```

Comment:
Could we refactor this test to be similar to the `ViewLabs` test where it checked the buttons on the scree rather than spying on the hook?